code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -69,12 +70,11 @@
 	var/visualAppearence = MAINTDRONE //What we appear as
 	var/hacked = FALSE //If we have laws to destroy the station
 	var/flavortext = \
-	"\n<big><span class='warning'>DO NOT INTERFERE WITH THE ROUND AS A DRONE OR YOU WILL BE DRONE BANNED</span></big>\n"+\
-	"<span class='notify'>Drones are a ghost role that are allowed to fix the station and build things. Interfering with the round as a drone is against the rules.</span>\n"+\
+	"<span class='notify'>Drones are a ghost role that are allowed to fix the station, build things, and serve the AI. Interfering with the round outside of AI orders as a drone is against the rules.</span>\n"+\
```

Comment:
If the AI interaction is the pinnacle of their existence the law should be moved closer to the top. If not, it shouldn't be the main focus of their description.